src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2444,6 +2451,41 @@ void ApplicationManagerImpl::HeadUnitReset(
   }
 }
 
+void ApplicationManagerImpl::ClearAppsPersistentData() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  const std::string apps_info_storage_file = get_settings().app_info_storage();
+  file_system::DeleteFile(apps_info_storage_file);
+
+  const std::string storage_folder = get_settings().app_storage_folder();
+  std::vector<std::string> files = file_system::ListFiles(storage_folder);
```

Comment:
@AByzhynar 
You could create typedef instead of typing std::vector<std::string> again and again